python/social_bot/tasks.py

```diff
@@ -259,7 +268,8 @@ def run(self):
                 agent_sentence = yield TeacherAction(
                     reward=reward, sentence="well done", done=False)
                 steps_since_last_reward = 0
-                self._move_goal(goal, loc)
+                if self._move_goal_during_episode:
```

Comment:
if not move_goal_during_episode, should we send a "done" here to end the episode?

Reply:
Rui is training pioneer to control the first goal (ball) to hit the other goal (blue_ball), with mutual information reward to encourage two balls to move together.  So reaching target shouldn't end episode.

Reply:
Done is not needed here because reaching target shouldn't end episode.